src/main/resources/explanations/challenge43.adoc

```diff
@@ -0,0 +1,8 @@
+=== Reddit Blunder
+
+Years ago, an error made by Microsoft developers in the way they set up the testing and debugging of the Windows 10 OS meant users could install
+any OS of their choice on Windows PCs and mobile devices. This was called Microsoft's 'golden key' blunder.
```

Comment:
Looking at the implementation of the challenge, i think it might be easier to leave this out. and instead say something like:
```suggestion
People easily make mistakes. They can for instance share an "innocent" piece of data over social media which later turns out to be a secret. Or they can just post something on the "wrong screen" and submit it.
```

What do you think @bendehaan ?

Reply:
I think we shouldn't reference the Microsoft case - I can't find evidence related to social media posts about the initial leak, so let's leave it at the social media post.